.gitignore

```diff
@@ -6,4 +6,20 @@ dockerfiles/run/**/keystore
 dockerfiles/run/**/glowroot
 .m2
 test-run.log
-build
\ No newline at end of file
+build
+
+*.iml
+.idea
+*.lib
+*.jar
+.floo
+.flooignore
+.m2
+_site/
+keystore
+derby.log
+src/homepage/Gemfile.lock
+mailbox/var/
+src/homepage/.sass-cache/
+KahaDB/
+.attach_pid*
```

Comment:
Nope. Set up a **global** git ignore for these files.

Could you document this as part as your "environment setup guide?"

Reply:
done it

Reply:
can u recreate the github link for my environment setup guide document, last one 404 not found so i can't write anything. Or should i create it on my own github?.